scripts/publish.sh

```diff
@@ -15,7 +15,10 @@ else
   TARGET_BRANCH=$BRANCH
 fi
 
-if ! ci-update-package --branch ${TARGET_BRANCH}; then
+if ci-update-package --branch ${TARGET_BRANCH};
+then
+  npm run generate-readme;
```

Comment:
Do we want to do this everytime we publish?

Reply:
we update package.json so we can update also the README.md , but we should push this to github only on promotion